src/controllers/SubredditsController.ts

```diff
@@ -0,0 +1,16 @@
+import { BodyParams, Controller, Post } from '@tsed/common';
+import { SubredditsCreation } from '../models/auth.types';
+
+import { SubredditsService } from '../services/SubredditsService';
+
+
+@Controller('/')
+export class SubredditsController {
+	constructor(private subredditsService: SubredditsService) {}
+
+	@Post('/subreddits/create')
+	// eslint-disable-next-line max-len
+	public async createSubreddit(@BodyParams() subreddits: SubredditsCreation): Promise<SubredditsCreation> {
+		return await this.subredditsService.create(subreddits);
+	}
```

Comment:
Is the eslint-disable necessary? I think few "Enter's" will do.

Reply:
Corrected with enter. Eslint activated.